### PR TITLE
Added test stubs for oscap hardening, disconnected to connected

### PR DESCRIPTION
**Pull request by kbidarkar**
features

This covers the following areas:

a) oscap hardening
b) disconnected to connected RedHat Repo syncs
c) oscap email reporting

### CODE REVIEW

**Comment by oshtaier** [comment]
Please, do not forget to use our naming conventions for your new tests (https://github.com/SatelliteQE/robottelo/issues/3083).

File: tests/foreman/ui/test_sync.py [hunk 1]
```diff
@@ -74,3 +80,30 @@ def test_positive_sync_rh_repos(self):
             sync = self.sync.sync_rh_repos(repos)
             # syn.sync_rh_repos returns boolean values and not objects
             self.assertTrue(sync)
+
+    @stubbed
+    @tier2
+    def test_disconnected_to_connected_sync_rh_repos(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_sync.py]: That test seems as tier3 as for me

**Comment by oshtaier** [comment]
Pending comments

**Comment by sthirugn** [comment]
@kbidarkar can you please send any requirements docs for both these features to my email? thank you.

**Comment by sthirugn** [comment]
@kbidarkar did you close this by accident?

**Pull request closed without merging**